### PR TITLE
removed banking_info column from users table, views and controllers

### PR DESCRIPTION
**Pull request by izuroxx**
#484 removed banking_info @carpodaster @ramonh be gentle ;p

### CODE REVIEW

**Comment by ramonh** [comment]
Looks good to me! 👍 Nice one

File: db/migrate/20160707112342_remove_banking_info_from_users.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class RemoveBankingInfoFromUsers < ActiveRecord::Migration
+  def change
+  	remove_column :users, :banking_info, :text
```
Comment (carpodaster) [inline db/migrate/20160707112342_remove_banking_info_from_users.rb]: Purely cosmetic stuff: the convention for indenting Ruby code is 2 spaces, no tabs. Your text editor can be set to enforce that, but the configuration options depend on the editor.

**Comment by carpodaster** [comment]
Yes, looks good! Thanks @izuroxx !

File: db/migrate/20160707112342_remove_banking_info_from_users.rb (hunk 1 above)
Comment (izuroxx) [inline db/migrate/20160707112342_remove_banking_info_from_users.rb]: sorry! i'll keep that in mind! :)

**Pull request merged**